WalletWasabi.Gui/Controls/TogglePasswordBox.cs

```diff
@@ -0,0 +1,61 @@
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using Avalonia.Data;
+using Avalonia.Styling;
+using System;
+using ReactiveUI;
+using System.Reactive.Linq;
+
+namespace WalletWasabi.Gui.Controls
+{
+	public class TogglePasswordBox : ExtendedTextBox, IStyleable
+	{
+		Type IStyleable.StyleKey => typeof(TogglePasswordBox);
+
+		public static readonly StyledProperty<bool> IsPasswordVisibleProperty =
+			AvaloniaProperty.Register<TogglePasswordBox, bool>(nameof(IsPasswordVisible), defaultBindingMode: BindingMode.TwoWay);
+
+		public bool IsPasswordVisible
+		{
+			get => GetValue(IsPasswordVisibleProperty);
+			set => SetValue(IsPasswordVisibleProperty, value);
+		}
+
+		public TogglePasswordBox()
+		{
+			UseFloatingWatermark = true;
+			Watermark = "Password";
+
+			this.GetObservable(IsPasswordVisibleProperty)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(x =>
+			{
+				IsPasswordVisible = x;
```

Comment:
Actually this can be https://github.com/zkSNACKs/WalletWasabi/blob/master/CONTRIBUTING.md#observableaspropertyhelpers-over-properties

Reply:
`IsPasswordVisible` property has to set `IsPasswordVisibleProperty` and that cannot be changed to `ObservableAsPropertyHelper`. This is the pattern to define `StyledPorperties`. 

http://avaloniaui.net/docs/authoring-controls/defining-properties#registering-styled-properties